lib/nerves_hub_cli/user.ex

```diff
@@ -14,7 +17,11 @@ defmodule NervesHubCLI.User do
           binary(),
           String.t()
         ) :: :ok | {:error, atom()}
-  def save_certs(pem_cert, pem_key, certificate_password) do
+  def save_certs(
+        <<"-----BEGIN", _::binary>> = pem_cert,
+        <<"-----BEGIN", _::binary>> = pem_key,
```

Comment:
Could you remove the pattern match here? If you feel strongly that this should be verified then make an `is_pem_format()` guard or similar.